src/components/apps/launch/ResourceRequirements.js

```diff
@@ -0,0 +1,357 @@
+                </ExpansionPanelSummary>
+
+                <ExpansionPanelDetails>
+                    <TableContainer component={Paper}>
+                        <Table>
+                            <TableBody>
+                                {!!min_cpu_cores && (
+                                    <TableRow>
+                                        <TableCell>
+                                            {getMessage("minCPUCores")}
+                                        </TableCell>
+                                        <TableCell>{min_cpu_cores}</TableCell>
+                                    </TableRow>
+                                )}
+                                {!!min_memory_limit && (
+                                    <TableRow>
+                                        <TableCell>
+                                            {getMessage("minMemory")}
+                                        </TableCell>
+                                        <TableCell>
+                                            {formatGBValue(min_memory_limit)}
+                                        </TableCell>
+                                    </TableRow>
+                                )}
+                                {!!min_disk_space && (
+                                    <TableRow>
+                                        <TableCell>
+                                            {getMessage("minDiskSpace")}
+                                        </TableCell>
+                                        <TableCell>
+                                            {formatGBValue(min_disk_space)}
+                                        </TableCell>
+                                    </TableRow>
+                                )}
+                            </TableBody>
+                        </Table>
+                    </TableContainer>
+                </ExpansionPanelDetails>
+            </ExpansionPanel>
+        )
+    );
+};
+
+/**
+ * Displays a table listing each step's resource requirements if its value is
+ * greater than 0.
+ *
+ * @param {Object} props
+ * @param {string} props.baseId
+ * @param {Object[]} props.requirements - Resource requirements for each step.
+ * @param {number} props.requirements[].step_number
+ * @param {number} props.requirements[].min_cpu_cores
+ * @param {number} props.requirements[].min_memory_limit
+ * @param {number} props.requirements[].min_disk_space
+ */
+const ResourceRequirementsReview = withI18N(({ baseId, requirements }) => {
+    const headerMessageKey =
+        requirements.length === 1
+            ? "resourceRequirements"
+            : "resourceRequirementsForStep";
+
+    return requirements.map((stepRequirements) => (
+        <StepResourceRequirementsReview
+            baseId={baseId}
+            key={stepRequirements.step_number}
+            stepRequirements={stepRequirements}
+            headerMessageKey={headerMessageKey}
+        />
+    ));
+}, messages);
```

Comment:
I kinda wish `withI18N` took `messages` as its first param 🤔 
If this looks too awkward to others, I can refactor my components out of the `withI18N` calls.

For example, this could become something like this:
```javascript
const ResourceRequirementsReview = withI18N(ResourceRequirementsReviewBase, messages);
```